### ISSUE TITLE
Tags - Background does not meet WCAG 2.1 requirements

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#8912]
Fluent tags do not meet WCAG 2.1 requirements for non-text contrast. They need to be 3:1 against background, but are currently 1.11:1 (Gray20).

**Comment by atneik** [comment]
@betrue-final-final what should be the new Gray to reach 3:1?

**Comment by betrue-final-final** [comment]
As there is content always in the item, so the plate isn't required as a contrast compliant hit target.
